### PR TITLE
Update xdmod-admin truncate to operate on tables that replaced modw.jobfacts

### PR DESCRIPTION
**Pull request by treydock**
## Description
Update `xdmod-admin` to truncate `modw.job_records` and `modw.job_tasks` in place of `modw.jobfacts` based on `xdmod-upgrade` output that `modw.jobfacts` is replaced by those two tables.

## Motivation and Context
To get truncating working on new tables.

## Tests performed
Ran truncation on test data in docker:

```shell
[root@4b309869edcc xdmod]# xdmod-admin -t -j -v --debug
2018-04-19 16:58:07 [info] Command: '/usr/bin/xdmod-admin' '-t' '-j' '-v' '--debug'
Truncate all job data? (yes/no): [no] yes
2018-04-19 16:58:08 [notice] Truncating job data
2018-04-19 16:58:08 [info] Truncating job tables in mod_shredder
2018-04-19 16:58:08 [info] Truncating shredded_job_pbs
2018-04-19 16:58:08 [info] Truncating shredded_job_sge
2018-04-19 16:58:08 [info] Truncating shredded_job_slurm
2018-04-19 16:58:08 [info] Truncating shredded_job_lsf
2018-04-19 16:58:08 [info] Truncating shredded_job
2018-04-19 16:58:08 [info] Truncating staging_job
2018-04-19 16:58:08 [info] Truncating job tables in mod_hpcdb
2018-04-19 16:58:08 [info] Truncating hpcdb_jobs
2018-04-19 16:58:08 [info] Truncating job tables in modw
2018-04-19 16:58:08 [info] Truncating modw.job_records
2018-04-19 16:58:08 [info] Truncating modw.job_tasks
2018-04-19 16:58:08 [info] Truncating modw.jobhosts
2018-04-19 16:58:08 [info] Truncating modw_aggregates.jobfact_by_year
2018-04-19 16:58:08 [info] Truncating modw_aggregates.jobfact_by_quarter
2018-04-19 16:58:08 [info] Truncating modw_aggregates.jobfact_by_month
2018-04-19 16:58:08 [info] Truncating modw_aggregates.jobfact_by_day

```

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: bin/xdmod-admin [hunk 1]
```diff
@@ -255,7 +255,8 @@ function truncateJobs()
         array(
             'db'     => $modwDb,
             'tables' => array(
-                'modw.jobfact',
+                'modw.job_records',
+                'modw.jobtasks',
```
Comment (plessbd) [inline bin/xdmod-admin]: I believe this is job_tasks

Comment (treydock) [reply bin/xdmod-admin]: Fixed.

**Comment by plessbd** [comment]
One more request, can you remove jobfactstatus, since that is no longer used with the new process.

**Comment by treydock** [comment]
Removed jobfactstatus.

**Pull request merged**